community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/main.tf

```diff
@@ -50,3 +52,6 @@ locals {
     network_storage = var.network_storage
   }
 }
+
+data "google_compute_default_service_account" "default" {
```

Comment:
We got to specify `project_id`, please add it as a variable to this module.

Reply:
Added.